### PR TITLE
Fix basic fee bugs

### PR DESCRIPTION
**Pull request by jsugarman**
Fix bugs that came in with Fee abstraction work 
- [x] basic fees all getting buttons to add dates attended erroneously
- [x] basic fee ordering in form is by description but should be by id

### CODE REVIEW

File: spec/models/fee/fixed_fee_type_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    describe 'default scope' do
+      before do
+        create(:fixed_fee_type, description: 'Ppppp')
+        create(:fixed_fee_type, description: 'Xxxxx')
+        create(:fixed_fee_type, description: 'Sssss')
+      end
+
+      it 'should order by description ascending' do
+        # expect(Fee::FixedFeeType.all.map(&:id)).to eq [1,3,2]
```
Comment (stephenrichards) [inline spec/models/fee/fixed_fee_type_spec.rb]: Better to remove the commented-out line

File: spec/models/fee/basic_fee_type_spec.rb [hunk 2]
```diff
@@ -2,12 +2,37 @@
 
 module Fee
   describe BasicFeeType do
+
     let(:fee_type)  { build :basic_fee_type }
+    DATES_ATTENDED_APPLICABLE_FEES = %w( BAF DAF DAH DAJ PCM SAF )
+    DATES_ATTENDED_NOT_APPLICABLE_FEES = %w( CAV NDR NOC PPE NPW )
```
Comment (aliuk2012) [inline spec/models/fee/basic_fee_type_spec.rb:8]: Why are these hard coded? Isn't there a risk that these dont match what is being used in actual code?

Comment (jsugarman) [inline spec/models/fee/basic_fee_type_spec.rb:8]: It is deliberately separating the app logic from the test so as to raise an error in the event of a change to the app that is not accounted for in the test

File: spec/models/fee/fixed_fee_type_spec.rb (hunk 1 above)
Comment (jsugarman) [inline spec/models/fee/fixed_fee_type_spec.rb]: done

**Pull request merged**